lib/SIL/Utils/PrunedLiveness.cpp

```diff
@@ -295,6 +296,13 @@ SimpleLiveRangeSummary PrunedLiveRange<LivenessWithDefs>::updateForDef(SILValue
       });
       break;
     }
+    case OperandOwnership::GuaranteedForwardingPhi: {
+      SILArgument *phi = cast<BranchInst>(use->getUser())
```

Comment:
I think the PhiOperand abstraction handles/hides all the BranchInst/DestBB stuff

Reply:
Fixed.